sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2926,7 +2928,7 @@ def __init__(
     self.validate()
 
     # We use this internal object ID to generate BigQuery export directories.
```

Comment:
Shall we update this code comment?

Reply:
Nothing changed really, the comment is still valid. I guess we can also include that this is used to create job names